docs/source/contributing.rst

```diff
@@ -472,9 +508,15 @@ Docs
 
 Staged docs for every new PR are built and accessible at ``soundata.github.io/preview/PR-<#PR_ID>`` in which ``<#PR_ID>`` is the pull request ID. 
```

Comment:
Staged docs are not built to `soundata.github.io/preview/PR-<#PR_ID>` anymore, they can be checked accessing through the `docs/readthedocs.org:soundata — Read the Docs build succeeded!` GitHub action. You have to go to "Details" and it will redirect you to an url like https://soundata--165.org.readthedocs.build/en/165/source/contributing.html (this is the one for this PR)